samples/nrf9160/lwm2m_client/overlay-assist-wifi.conf

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Overlay to use nRF7002 EK on top of nrf9160 DK for Wi-Fi scanning
+
+# Disable modem traces as UART1 is disabled
+CONFIG_NRF_MODEM_LIB_TRACE=n
+
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+#LwM2M Client utility library options
+CONFIG_LWM2M_CLIENT_UTILS_GROUND_FIX_OBJ_SUPPORT=y
+CONFIG_LWM2M_CLIENT_UTILS_VISIBLE_WIFI_AP_OBJ_SUPPORT=y
+CONFIG_LWM2M_CLIENT_UTILS_WIFI_AP_SCANNER=y
+CONFIG_LWM2M_CLIENT_UTILS_LOCATION_ASSIST_EVENTS=y
+#Enable LwM2M 1.1
+CONFIG_LWM2M_VERSION_1_1=y
+CONFIG_LWM2M_COMPOSITE_PATH_LIST_SIZE=10
+#Enable SenML-CBOR content format
+CONFIG_LWM2M_RW_SENML_CBOR_SUPPORT=y
+CONFIG_ZCBOR=y
+CONFIG_ZCBOR_CANONICAL=y
+CONFIG_LWM2M_RW_SENML_CBOR_RECORDS=65
+# Increase AT monitor heap because %NCELLMEAS notifications can be large
+CONFIG_AT_MONITOR_HEAP_SIZE=512
+
+# Does not work with buttons&LEDs
+CONFIG_APP_PUSH_BUTTON=n
+CONFIG_APP_ONOFF_SWITCH=n
+CONFIG_DK_LIBRARY=n
+
+# Not for LTE throughput testing
+CONFIG_NRF_MODEM_LIB_SHMEM_TX_SIZE=4096
+CONFIG_NRF_MODEM_LIB_SHMEM_RX_SIZE=4096
+
+# Actual configs for the Wi-Fi
+CONFIG_WIFI=y
+CONFIG_NRF700X_REV_A=y
+CONFIG_WIFI_NRF700X=y
+CONFIG_NET_L2_WIFI_MGMT=y
+
+#CONFIG_NET_L2_WIFI_SHELL=y
+
+# System settings
+CONFIG_NEWLIB_LIBC=y
+
+# Scan only using offload API
+CONFIG_WPA_SUPP=n
+
+# For nRF9160 the default is socket interface
+CONFIG_NET_DEFAULT_IF_ETHERNET=y
+CONFIG_MBEDTLS=n
+CONFIG_NORDIC_SECURITY_BACKEND=n
+
+# Networking
+CONFIG_NET_L2_ETHERNET=y
+#CONFIG_NET_SHELL=y
+CONFIG_NET_NATIVE=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_DHCPV4=y
+CONFIG_NET_STATISTICS=y
```

Comment:
Do we really need full IP stack for just scanning?

Reply:
Not actually sure, I'll try to clean some of the configs from the overlay.

Reply:
Cleaned the overlay file. There are currently some issues with the external flash when using the EK-board over nrf91, both are using the same spi and causes some issues.